sh/openrc-run.sh.in

```diff
@@ -39,6 +39,22 @@ sourcex()
 	fi
 }
 
+apply_ulimits()
+{
+	if [ $(( $# % 2 )) -eq 0 ]; then
```

Comment:
we could probably invert this check:

```suggestion
	if [ $(( $# % 2 )) -eq 1 ]; then
		eerror "${RC_SVCNAME}: Invalid RC_ULIMIT setting"
		return 1
	fi
```

so that the whole function has less unnecessary indentation